src/taxonomy/factory.ts

```diff
@@ -66,7 +65,7 @@ export const TaxonFactory: Factory<Taxon, TaxonFactoryParams> = {
 
 // VernacularNameFactory
 export type VernacularNameParams = {
-  lang: LanguageCode
+  lang: string
```

Comment:
language code is in ISO 639-3 format (3 characters) not in the 639-2, like in LanguageCode enum.